readme.md

```diff
@@ -217,6 +217,7 @@ Made with Electron.
 - [Activity Monitor](http://electron.atom.io/blog/2017/01/19/simple-samples#activity-monitor) - Shows a doughnut chart of the CPU system, user, and idle activity time.
 - [Hash](http://electron.atom.io/blog/2017/01/19/simple-samples#hash) - Shows the hash values of entered text using different algorithms.
 - [Prices](http://electron.atom.io/blog/2017/01/19/simple-samples#prices) - Shows the current price of oil, gold, and silver using the Yahoo Finance API.
+- [TouchBar API Sample](https://github.com/Rawnly/electron-touchbar-example) - Sample touchbar integration.
```

Comment:
It's `Touch Bar`, not `TouchBar` or `touchbar`. The word `Sample` is also moot as it's in the Sample section.

Reply:
👍